modules/core/service/main.tf

```
@@ -188,6 +266,62 @@ resource "aws_ecs_service" "main_ignore_desired_count_changes" {
     }
   }
 
+  dynamic "service_connect_configuration" {
+    for_each = var.service_connect_configuration == null ? [] : [var.service_connect_configuration]
+
+    content {
+      enabled = lookup(service_connect_configuration.value, "enabled", null)
+
+      dynamic "log_configuration" {
+        for_each = var.service_connect_configuration.log_configuration == null ? [] : [var.service_connect_configuration.log_configuration]
+
+        content {
+          log_driver = lookup(log_configuration.value, "log_driver", null)
+          options    = lookup(log_configuration.value, "options", null)
+
+          dynamic "secret_option" {
+            for_each = log_configuration.value.secret_option == null ? [] : [log_configuration.value.secret_option]
+
+            content {
+              name       = secret_option.value.name
+              value_from = secret_option.value.value_from
+            }
+          }
+        }
+      }
+
+      namespace = lookup(service_connect_configuration.value, "namespace", null)
+
+      dynamic "service" {
+        for_each = var.service_connect_configuration.service == null ? [] : [var.service_connect_configuration.service]
```

Service is a [list](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/ecs_service#service_connect_configuration) 
```suggestion
        for_each = service_connect_configuration.service
```